src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorFeatures.kt

```diff
@@ -457,6 +458,10 @@ class GardenVisitorFeatures {
         if (GardenAPI.inGarden() && config.hideChat && hideVisitorMessage(event.message)) {
             event.blockedReason = "garden_visitor_message"
         }
+
+        partialAcceptedPattern.matchMatcher(event.message) {
+            LorenzUtils.chat("Open visitor inventory again to update the number of items needed!")
```

Comment:
Maybe use the phrase "talk to the spaceman again" instead

Reply:
How about "talk to the visitor again"? I think it's better not to hardcode it in case Hypixel adds another Spaceman-like visitor in the future.